vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -169,11 +188,23 @@ fire_and_forget WinRTWebSocketResource::PerformPing()
       CheckedReinterpretCast<const uint8_t*>(s.c_str()) + s.length());
     self->m_writer.WriteBytes(arr);
 
-    co_await self->m_writer.StoreAsync();
+    auto async = self->m_writer.StoreAsync();
+
+    co_await lessthrow_await_adapter<DataWriterStoreOperation>{async};
 
-    if (self->m_pingHandler)
+    if (SUCCEEDED(async.ErrorCode()))
+      {
```

Comment:
fix indentation (curious why yarn format didn't catch this?)

Reply:
Looks like we're suppressing Clang format for this file? I thought we weren't going to do that, since style here doesn't match the rest of our code.